config/default.settings.local.php

```diff
@@ -13,18 +13,18 @@
  */
 
 // Database settings.
-$databases = array (
+$databases = array(
   'default' =>
-    array (
+    array(
```

Comment:
I'm conflicted on that one. _Technically_ it [isn't a drupal standard](https://www.drupal.org/coding-standards#array) but we have been using it...